ui/clubhouse.js

```diff
@@ -902,7 +902,7 @@ var Component = GObject.registerClass({
                     g_object_path: this._clubhousePath,
                     g_flags: Gio.DBusProxyFlags.NONE,
                 });
-                this.proxy.init(null);
+                await this.proxy.init_async(GLib.PRIORITY_DEFAULT, null);
```

Comment:
Not sure if necessary but may be good to pass a cancellable and cancel it on `disable()` in case disable is invoked before the proxy is ready.